types/src/v18/network/mod.rs

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: CC0-1.0
+
+//! The JSON-RPC API for Bitcoin Core `v0.18` - network.
+//!
+//! Types for methods found under the `== Network ==` section of the API docs.
+//!
+/// These types do not implement `into_model` because apart from fee rate there is no additional
+/// `rust-bitcoin` types needed.
+use serde::{Deserialize, Serialize};
```

Comment:
nit: newline please.
```suggestion

use serde::{Deserialize, Serialize};
```

Reply:
Done

Reply:
This failed format CI with the space. I copied the file from v17 which has no newline after, but the commenting changes from `//!` to `///`. I have now changed this file to use the module comments, which cargo format does want a new line after.

Reply:
Oh I did not notice the weird comment `///` thing. I've done #158, I'll merge that then you can rebase on master and exclude the comment all together here as well.